src/css/_buttons.scss

```diff
@@ -0,0 +1,50 @@
+@import 'mixins', 'variables';
```

Comment:
Unsure why it's necessary for @import to be utilized here when base.scss has them. For now, it's functional.